Makefile

```diff
@@ -10,3 +10,6 @@ serve: ## Run the book dev server
 .PHONY: help
 help:
 	@cat $(MAKEFILE_LIST) | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+writing-gleam/command-line-reference.md: writing-gleam/command-line-reference.sh
```

Comment:
Can we have `make` call this automatically? 🙏

Reply:
Sure can, the Makefile was one of the things I didn't want to touch since I don't know how/when this repo is built.

Reply:
Actually can we inline the calling of the script into the `build` command? I think this current one will make it so that we have to `touch writing-gleam/command-line-reference.sh` to update the page as make will think it has already been build.

Reply:
It is now inlined! I noticed it was kinda slow so debugged and cut down on the number of `gleam help` calls. If we're gonna do it every time might as well not waste time.